.gitignore

```diff
@@ -123,7 +123,7 @@ fabric.properties
 
 # Icon must end with two \r
 Icon
-
+!.src/components/ui/icon/
```

Comment:
Why is this in the PR?

Reply:
I might be wrong, and recheck this but my editor was giving a message that some files were included that were supposed to be ignored according to the .gitignore. This could only be this file since the name is matching, after adding this it worked like a charm. But I will check again!

Reply:
Removed.